test/unstructured_storage.js

```diff
@@ -0,0 +1,61 @@
+const getContract = name => artifacts.require(name)
+
+contract('Constants', accounts => {
+  let app, kernel
+
+  beforeEach(async () => {
+    app = await getContract('AppStubStorage').new()
+    kernel = await getContract('Kernel').new()
+  })
+
+  it('test', async () => {
+    const appId = '0x1234000000000000000000000000000000000000000000000000000000000000'
+    const pinnedCode = '0x1200000000000000000000000000000000005678'
+
+    // set values
+    await app.initialize()
+    await app.setKernelExt(kernel.address)
+    await app.setAppIdExt(appId)
+    await app.setPinnedCodeExt(pinnedCode)
+
+    //checks
+    assert.equal(
+      parseInt(await web3.eth.getStorageAt(app.address, (await app.getInitializationBlockPosition())), 16),
```

Comment:
I would get the positions from `test/mocks/KeccakConstants.sol` rather than from 'AragonApp', even though that equality already tested in `test/keccak_constants.js`. Would allow to remove all the position public getters from `AppStubStorage`.

Reply:
Well, but if we remove the public getters from `AppStubStorage` then tests in `keccak_constants.js` won't work. Here I see we are testing 2 things:
- That hardcoded position is what it is intended to be (not a big deal, but better to have it, to avoid collisions)
- That things are properly stored at those positions.
Testing the second one first one would be implicitly tested if we use positions from `KeccaKConstants` but I think it's better to have both for 2 reasons: 1) in case of failing, it's eaiser to know what the problem is, 2) we could be pointing at another position that has the same value (yes, unlikely, but still) and having both tests we would have more chances to catch it.